server/src/main/java/org/elasticsearch/snapshots/SnapshotShardsService.java

```diff
@@ -423,77 +392,6 @@ private void syncShardStatsOnNewMaster(ClusterChangedEvent event) {
         }
     }
 
-    /**
-     * Internal request that is used to send changes in snapshot status to master
-     */
-    public static class UpdateIndexShardSnapshotStatusRequest extends MasterNodeRequest<UpdateIndexShardSnapshotStatusRequest> {
```

Comment:
This stuff all got moved to `SnapshotsService` exactly as is without changes

Reply:
Now it is used in different services it could maybe be located in its own file